### PR TITLE
Feature add new tests to api

### PR DESCRIPTION
**Pull request by flaminestone**

### CODE REVIEW

File: asserts/js/xlsx/smoke/api/create_color_by_name.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+builder.CreateFile("xlsx");
+var oWorksheet = Api.GetActiveSheet();
+oColor = Api.CreateColorByName("ltSteelBlue");
+oWorksheet.GetRange("A2").SetValue("Text with color");
+oWorksheet.GetRange("A2").SetFontColor(oColor);
+builder.SaveFile("xlsx", "CreateColorByName.xlsx");
+builder.CloseFile();
```
Comment (JohnS42) [inline asserts/js/xlsx/smoke/api/create_color_by_name.js]: newline?

File: spec/xlsx/smoke/api_range_spec.rb [hunk 2]
```diff
@@ -63,4 +63,75 @@
...
+    expect(xlsx.worksheets.first.rows[1].cells[0].raw_text).to eq('This is the cell with a color set to its background')
+    expect(xlsx.worksheets.first.rows[3].cells[0].style.fill_color.color).to eq(OoxmlParser::Color.new())
+    expect(xlsx.worksheets.first.rows[3].cells[0].raw_text).to eq('This is the cell with a default background color')
+  end
+
+  it 'ApiRange | SetNumberFormat method' do
+    pending 'https://github.com/ONLYOFFICE/ooxml_parser/issues/317'
+    formats = ["General", "0.00", "$#,##0.00",  "_($* #,##0.00_)",  "m/d/yyyy", "[$-F800]dddd, mmmm dd, yyyy", "[$-F400]h:mm:ss AM/PM", "0.00%", "0.00%", "# ?/?", "0.00E+00", "Text"]
+    xlsx = DocBuilderWrapper.new.build_doc_and_parse('asserts/js/xlsx/smoke/api_range/set_number_format.js')
+    12.times do |i|
```
Comment (ShockwaveNN) [inline spec/xlsx/smoke/api_range_spec.rb]: formats.each_with_index

File: spec/xlsx/smoke/api_range_spec.rb [hunk 3]
```diff
@@ -63,4 +63,75 @@
...
+    expect(xlsx.worksheets.first.rows[1].cells[0].style.borders.right.color).to eq(nil)
+    expect(xlsx.worksheets.first.rows[1].cells[0].style.borders.top.color).to eq(nil)
+  end
+
+  it 'ApiRange | SetWrap method' do
+    xlsx = DocBuilderWrapper.new.build_doc_and_parse('asserts/js/xlsx/smoke/api_range/set_wrap.js')
+    expect(xlsx.worksheets.first.rows[1].cells[0].style.alignment.wrap_text).to be_truthy
+    expect(xlsx.worksheets.first.rows[1].cells[2].style.alignment.wrap_text).to be_falsey
+  end
+end
```
Comment (JohnS42) [inline spec/xlsx/smoke/api_range_spec.rb]: newline?

Comment (ShockwaveNN) [inline spec/xlsx/smoke/api_range_spec.rb]: ouch
image

File: asserts/js/xlsx/smoke/api/create_color_by_name.js (hunk 1 above)
Comment (flaminestone) [inline asserts/js/xlsx/smoke/api/create_color_by_name.js]: This is a new example style of code. Rubocop is not detected it.

Comment (flaminestone) [inline asserts/js/xlsx/smoke/api/create_color_by_name.js]: Because rubocop ignore .js files

File: spec/xlsx/smoke/api_range_spec.rb (hunk 2 above)
Comment (flaminestone) [inline spec/xlsx/smoke/api_range_spec.rb]: fixed

**Pull request merged**